### PR TITLE
Feature/klant betrokkene

### PR DESCRIPTION
**Pull request by annashamray**
Schema: https://github.com/VNG-Realisatie/gemma-zaken/issues/1323#issuecomment-540012153
Issue: https://github.com/VNG-Realisatie/gemma-zaken/issues/1366

### CODE REVIEW

**Review by joeribekker — DISMISSED** [review]
I think a rebase will remedy the issues

**Comment by annashamray** [comment]
> I think a rebase will remedy the issues

the branche is rebased

File: src/kcc/datamodel/models.py [hunk 1]
```diff
@@ -18,6 +22,18 @@ class Klant(APIMixin, models.Model):
     adres = models.CharField(max_length=1000, blank=True)
     telefonnummer = models.CharField(max_length=20, blank=True)
```
Comment (stevenbal) [inline src/kcc/datamodel/models.py]:
```suggestion
    telefoonnummer = models.CharField(max_length=20, blank=True)
```

File: src/kcc/api/tests/test_klant.py [hunk 2]
```diff
@@ -72,6 +308,115 @@ def test_update_klant(self):
...
+        self.assertEqual(adres.wpl_woonplaats_naam, "New place")
+
+        buitenland.refresh_from_db()
+        self.assertEqual(buitenland.lnd_landnaam, "New land")
+
+    def test_update_klant_vestiging(self):
+        klant = KlantFactory.create(betrokkene_type=KlantType.vestiging)
+        detail_url = reverse(klant)
+
+        response = self.client.patch(
```
Comment (stevenbal) [inline src/kcc/api/tests/test_klant.py:361]: you could maybe add tests for `PUT` as well, just to be sure

**Review by stevenbal — CHANGES_REQUESTED** [review]

File: src/kcc/api/tests/test_klant.py (hunk 2 above)
Comment (annashamray) [reply src/kcc/api/tests/test_klant.py:361]: tests for `PUT` are added

**Review by stevenbal — APPROVED** [review]

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**